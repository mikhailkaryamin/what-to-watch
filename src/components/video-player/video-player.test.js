import React from 'react';
import renderer from 'react-test-renderer';

import VideoPlayer from './video-player.jsx';

describe(`Video player`, () => {
  const PREVIEW_VIDEO_LINK = `https://upload.wikimedia.org/wikipedia/commons/transcoded/8/87/Schlossbergbahn.webm/Schlossbergbahn.webm.480p.vp9.webm`;
  const POSTER_IMAGE = `img/bg-the-grand-budapest-hotel.jpg`;

  const wrapper = renderer
    .create(<VideoPlayer
      previewVideoLink={PREVIEW_VIDEO_LINK}
      posterImage={POSTER_IMAGE}
    />);

  test(`should render video player`, () => {
    expect(wrapper).toMatchSnapshot();
  });
});
