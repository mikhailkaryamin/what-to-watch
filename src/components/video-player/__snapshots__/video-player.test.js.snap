// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Video player should render video player 1`] = `
<video
  autoPlay={true}
  muted={true}
  poster="img/bg-the-grand-budapest-hotel.jpg"
  src="https://upload.wikimedia.org/wikipedia/commons/transcoded/8/87/Schlossbergbahn.webm/Schlossbergbahn.webm.480p.vp9.webm"
/>
`;
